orttraining/orttraining/test/python/orttraining_test_utils.py

```diff
@@ -123,7 +123,7 @@ def run_test(model, model_desc, device, args, gradient_accumulation_steps, fp16,
             allreduce_post_accumulation=allreduce_post_accumulation,
             get_lr_this_step=get_lr_this_step if use_internal_get_lr_this_step else None,
             loss_scaler=loss_scaler if use_internal_loss_scaler else None,
-            _opset_version=12,
+            _opset_version=14,
```

Comment:
shall we let this be the default setting?

Reply:
The value was explicitely set even though the default opset was 12. I did not change that.